### PR TITLE
Update course plan

### PR DESCRIPTION
**Pull request by denstorti**
# Description

Update class plan.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [x] Documentation update
- [x] Repo management

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,36 +99,56 @@ The content can be accessed through each specific README link below.
...
+    - AWS RDS (Relational Database Service)
+    - Project #1 Kick-off
 - [Class 05](classes/05class/README.md)
     - CI/CD Concepts
+      - AWS CodeBuild
     - Pipelines
-    - 3 Musketeers
+      - AWS CodePipeline
+      - AWS CodeDeploy (with Terraform)
+      - Github Actions ??
```
Comment (denstorti) [inline README.md]: Any comments on Github Actions? @kikobr82 @caiocezart

Comment (kikobr82) [reply README.md]: I have played with it recently at work..
We can use it as a CI/CD tool.. it seems a good option ...

File: README.md [hunk 2]
```diff
@@ -99,36 +99,56 @@ The content can be accessed through each specific README link below.
...
     - AWS CLI
-    - AWS IAM
-    - AWS S3
-    - AWS RDS
-    - IaC - Terraform
+    - AWS S3 (Simple Storage Service)
+      - Website
+      > - AWS Cloudfront
+      > - AWS WAF (Web Application Firewall)
     - MakeFile
```
Comment (kikobr82) [inline README.md]: Makefile is now on class05

File: README.md [hunk 3]
```diff
@@ -99,36 +99,56 @@ The content can be accessed through each specific README link below.
...
+    - AWS IAM (Identity and Access Management)
     - AWS CLI
-    - AWS IAM
-    - AWS S3
-    - AWS RDS
-    - IaC - Terraform
+    - AWS S3 (Simple Storage Service)
+      - Website
+      > - AWS Cloudfront
+      > - AWS WAF (Web Application Firewall)
```
Comment (kikobr82) [inline README.md]: Cloudfront and waf is not on class04 anymore

File: README.md (hunk 1 above)
Comment (kikobr82) [reply README.md]: Let's move on with GitHub Actions. we can have the following for class05:
- 3 Musketeers
 - Docker
 - Make
 - Compose
- CI/CD Concepts
  - GitHub Actions

**Pull request merged**